STAR.Web/Views/Contractor/Details.cshtml

```diff
@@ -3,86 +3,130 @@
 
-                          // remove the current input
-                          terms.pop();
-                          // add the selected item
-                          terms.push(ui.item.label);
+                              // remove the current input
+                              terms.pop();
+                              // add the selected item
+                              terms.push(ui.item.label);
 
-                          // get the ids - do it here before the placeholder is added
-                          var skillIds = terms.
-                            map(function (s) { return skills.find(function (sk) { return sk.name.toLowerCase() === s.trim().toLowerCase(); }).id; });
-                          $("#skillIds").val(skillIds);
+                              // get the ids - do it here before the placeholder is added
+                              var skillIds = terms.
+                                map(function (s) { return skills.find(function (sk) { return sk.name.toLowerCase() === s.trim().toLowerCase(); }).id; });
+                              $("#skillIds").val(skillIds);
 
+                              // add placeholder to get the comma-and-space at the end
+                              terms.push("");
+                              this.value = terms.join(", ");
 
-                          // add placeholder to get the comma-and-space at the end
-                          terms.push("");
-                          this.value = terms.join(", ");
-
-                          return false;
-                      }
-                  });
+                              return false;
+                          }
+                      });
+                });
         });
-    });
-</script>
+    </script>
 }
 
-@using (Html.BeginForm("Details", "Contractor", new { id = Model.ID }, FormMethod.Post, new { @class = "form-horizontal" })) {
+@using(Html.BeginForm("Details", "Contractor", new { id = Model.ID }, FormMethod.Post, new { @class = "form-horizontal" })) {
     @Html.AntiForgeryToken()
-
-    <div class="form-group">
-        @Html.LabelFor(m => m.FirstName)
-        @Html.TextBoxFor(m => m.FirstName, new { @class = "form-control" })
-    </div>
-    <div class="form-group">
-        @Html.LabelFor(m => m.LastName, new { @class = "control-label" })
-        @Html.TextBoxFor(m => m.LastName, new { @class = "form-control" })
-    </div>
-    <div class="form-group">
-        <label for="txtSearch" class="control-label">Skills: </label>
-        @Html.TextBox("txtSearch", Model.Skills.Select(s => s.Name).Delimit(), new { id = "txtSearch", @class = "form-control" })
-        @Html.Hidden("skillIds", Model.Skills.Select(s => s.SkillId).Delimit(), new { id = "skillIds" })
-    </div>
-    <input class="btn btn-default" type="submit" value="Save" />
-    @Html.ActionLink("Cancel", "Index", new { @class="btn btn-default" })
-}
\ No newline at end of file
+    <form name="updateContractor">
```

Comment:
`@Html.BeginForm(...)` emits a form tag to the output stream.  Having a form tag here will create a second one.  If there is need for the name attribute, it can be done using the `BeginForm` method.